cmd/capdctl/main.go

```diff
@@ -116,6 +118,8 @@ func main() {
 	case "machine-deployment":
 		machineDeployment.Parse(os.Args[2:])
 		fmt.Fprint(os.Stdout, machineDeploymentYAML(machineDeploymentOpts))
+	case "version":
+		fmt.Println(versioninfo.VersionInfo(capdctl))
```

Comment:
This println prints one too many new lines. Change this to a bare Print() please:

```
root@ip-172-31-23-191:~/cluster-api-provider-docker# capdctl version
capdctl version info:
GitReleaseTag: "", Major: "", Minor: "", GitRelaseCommit: ""
Git Branch: ""
Git commit: ""
Git tree state: ""

root@ip-172-31-23-191:~/cluster-api-provider-docker# ./
```